src/ccalls.jl

```diff
@@ -183,4 +185,4 @@ end
 
 # For backwards compatibility
 # TODO: remove this again
-GAPFuncs = Globals
+const GAPFuncs = Globals
```

Comment:
Is there any code out there still using `GAPFuncs`? I guess @CarloSircana uses it in `CarloLibGAPTest`, but adjusting that is a simple matter of search and replace, no?

Reply:
Well, I am not sure. We can just remove it and see if anybody complains ;)